app/src/main/java/com/mikhailovskii/weatherandroid/ui/choose_location/ChooseLocationActivity.kt

```diff
@@ -0,0 +1,51 @@
+package com.mikhailovskii.weatherandroid.ui.choose_location
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mikhailovskii.weatherandroid.R
+import com.mikhailovskii.weatherandroid.ui.main.MainActivity
+import kotlinx.android.synthetic.main.activity_choose_location.*
+
+class ChooseLocationActivity : AppCompatActivity(), ChooseLocationContract.ChooseLocationView {
+
+    private val presenter = ChooseLocationPresenter()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_choose_location)
+
+        presenter.attachView(this)
+
+        next_btn.setOnClickListener {
+            if (country_tiet.text != null && city_tiet.text != null) {
```

Comment:
@SergeiMikhailovskii don't you think that black and null is basically the same, so they should be handled by the same case in your `when` statement?